### DISCUSSION TITLE
Using terminalGPT with a codebase

### DISCUSSION THREAD

**Discussion by SuperElectron** [jucasoliveira/terminalGPT#28] [General]
Hello,
First off, absolutely love this! Way to go and I am excited to see the progression of your developments ... and possible future integrations with IDEs like VScode and Jetbrains products.

I was searching around for docs/manuals/wiki etc and realize that this is quite new; however, I would like to gain a better understanding of how to use this with codebases - and in particular privately hosted code bases.

**Comment by jucasoliveira** [comment]
Hi, @SuperElectron there's no intrinsic integration yet. I'm creating modules where we can use openAI in a meaningful way to chat on the terminal, without the need to go to ChatGPT.

With that said, the next step of the project will be to really integrate with the code base. The idea is too broad yet and I need to clarify what's actually needed by the community. We can create modules where you can run code analysis for your codebase, or use it as a chatbot service for your server. 

How do you think you'd like to see this project go? 


Thanks
